components/Form/Form.js

```diff
@@ -0,0 +1,41 @@
+import {
+  StyledButton,
+  StyledForm,
+  StyledInput,
+  StyledLabel,
+} from "./Form.styled";
+export default function Form({ onSubmit /*state = {} */ }) {
```

Comment:
```suggestion
export default function Form({ onNewCardCreation }) {
```
always delete comments (and unnecessary console.logs) before pushing your code :)